src/components/common/links/external-link.tsx

```diff
@@ -10,6 +10,16 @@ import type { IconName } from '../fork-awesome/types'
 import { ShowIf } from '../show-if/show-if'
 import type { LinkWithTextProps } from './types'
 
+/**
+ * An external link.
```

Comment:
An explanation "what is an external link" would be nice. Or anything that explains why you should use this component if you link to external services.